koocook_core/models/review.py

```diff
@@ -1,20 +1,34 @@
+from typing import Union
 from django.db import models
-from django.core.exceptions import ValidationError
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
 from django.utils.translation import gettext_lazy as _
 from .base import SerialisableModel
 
 from ..support import FormattedField
 from .base import SerialisableModel
 
-__all__ = ['Comment', 'Rating', 'AggregateRating']
+__all__ = ('AggregateRating', 'Comment', 'Rating', 'ReviewableModel')
 
 
-def create_empty_aggregate_rating(**kwargs) -> 'AggregateRating':
-    """Creates an empty aggregate rating"""
-    return AggregateRating.objects.create(rating_value=0, rating_count=0, **kwargs)
+class ReviewableModel:
+    """
+        This creates an AggregateRating only after reviewables are actually created
+        and saved to a database; that is, it will be created once in need, not always.
+    """
```

Comment:
The docstring has one too many indent

Reply:
It's a visual indent. But for the convention's sake, I'll change it then.